### ISSUE TITLE
Is it possible to keep camera orientation from free mouse rotation inside MagicaVoxel to an exported svg sprite ?

### ISSUE THREAD

**Issue by niloct** [Arlorean/Voxels.Core#1]
Thanks for your tool.

I'm trying to export a specific camera setup from MagicaVoxel after free rotating with mouse, to an SVG sprite with `Voxels.CommandLine.exe`.

Is this possible ? The SVG file always keep same orientation.

I'm getting .vox sprites from this awesome site: https://rooms.xyz

I appreciate your help.

**Comment by niloct** [comment]
[yorkshire.zip](https://github.com/Arlorean/Voxels.Core/files/11436696/yorkshire.zip)

It seems the orientation you set on MagicaVoxel is never exported to the .vox format, according to this: https://github.com/ephtracy/voxel-model/issues/12

**Comment by Arlorean** [comment]
Did you just essentially want a way to rotate it 90 degrees so the dog is facing the camera?
![image](https://github.com/Arlorean/Voxels.Core/assets/1547800/1d2d544c-41c4-498b-9516-e6afd4fe3918)

**Comment by niloct** [comment]
Yes, but for instance I have another image that would be facing the camera from the left.
Would it be possible to define the angle as a parameter to the command line tool ?

**Comment by Arlorean** [comment]
I've been looking at this but the arbitrary angle causes the problem of what order to draw the faces of the voxels. If we were using a 3D engine, with a z-buffer, we could just throw all the quads at it and get the right result, but with SVG we have to do all that face ordering in software. It's doable but not a simple case of changing the matrix rotation angles and you're done.

**Comment by Arlorean** [comment]
I just had it all working, went to commit it and got a "detached head" message. Not sure what I was doing I followed an online tutorial to "fix" it. Long story short I've just lost all my changes. I'll see if I can get motivation to do this again at some point but for now I just feel beaten by git. Sorry.

**Comment by niloct** [comment]
I really know that pain. Well if it's a comfort everything I ever lost in the same way I got a better coded version later that would not benefit from having the old version restored.
